Architecture.md

```diff
@@ -36,17 +43,36 @@
 
 #### CI/CD
 
-***TODO: [UNIFIED WORK] describe use of Github Actions as means of CI/CD***
+Each repository has a GitHub Actions CI to execute format check, linter and tests at each push. This ensures a good code quality and a good test coverage for our entire project.
+
+Most projects use our own Actions available in a [central repository](https://github.com/paastech-cloud/.github). 
+This ensures that all projects have the same CI/CD pipeline and that the CI/CD pipeline is easily maintainable.
+Moreover, thanks to Rust easy documentation creation process, a simple CI job is able to build the documentation and deploy it on [GitHub Pages](https://pages.github.com/) for every Rust project.
+Furthermore, at a tag push on main, or on a workflow call, a Docker image is built for every project that has this workflow set up.
+This image is then pushed as a repository package to the [GitHub Container Registry](https://ghrc.io) and is available for use.
 
+On top of that, the web UI is automatically deployed on [Github Pages](https://pages.github.com/).
+
+#### Deployment
```

Comment:
Add a line return between the title and the paragraph

Reply:
isn't it there already ?